tests/templates/test_state_preparations.py

```diff
@@ -359,6 +389,97 @@ def circuit(state_vector):
             state_vector = np.array([0, 2, 0, 0])
             circuit(state_vector)
 
+    # fmt: off
+    @pytest.mark.parametrize("state_vector, n_wires", [
+        ([.5,.5,.5,.5],2),
```

Comment:
Just a reminder for `black -l 100` still recommended.

Reply:
Is this regarding `@pytest.mark.parametrize...` ? I ran black -l 100 a few times without any change. Probably because we have ` # fmt: off` I could edit this manually.

Reply:
Yes I just noticed that formatting is off for this block.
```suggestion
        ([.5, .5, .5, .5], 2),
```
Nothing major though!